charts/vald/templates/index/job/correction/cronjob.yaml

```diff
@@ -28,7 +28,7 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/version: {{ .Chart.Version }}
 spec:
-  schedule: {{ $corrector.schedule }}
+  schedule: {{ $corrector.schedule | quote }}
```

Comment:
> **NOTE**
I added the `quote` syntax because if the following values are passed, an parse error will occur.
- user value: `"*/1 * * * *"`
- error: `Error: YAML parse error on vald/templates/index/job/save/cronjob.yaml: error converting YAML to JSON: yaml: line 28: did not find expected alphabetic or numeric character`

Reply:
👉 https://github.com/helm/helm/issues/9917#issuecomment-875569096